court/models.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+
+
+class Court(models.Model):
+    id = models.IntegerField(primary_key=True)
+    x = models.FloatField(default=0)
+    y = models.FloatField(default=0)
```

Comment:
What's the reason to have a default value here?

Reply:
When I wrote the default value, I was thinking about some habits from other courses. 
As for this PR, the mandatory data members are ID, X, Y, Max Players (city and neighborhood could have been blank).
So I was thinking that each mandatory field should have a default value, just because of that habit from other courses. I'm not saying that it's the best practice, but what is your opinion about default values in data members in general? Should we use them, or should we not?

Reply:
I think default values should be used when it makes sense. Coordinates is something that is expected to be unique, so having a default value to it doesn't make sense to me.

In general, default values have the potential for introducing ambiguity, and ambiguity is something we're trying to avoid.

Reply:
Thank you for your comment. I have fixed this part.